deploy.sh

```diff
@@ -363,6 +373,9 @@ case "$1" in
   deploy_unleash_importer_image)
     deploy_unleash_importer_image
     ;;
+  add_group_role_to_rbac)
```

Comment:
Should this target also be added to the `deploy_with_hbi_demo` target (along with the other seeding functions)? The idea is that one target can deploy everything and give you good indicative data for demos.

Reply:
Could it be hard to define what kind of permissions needs to be in the demo, but I can add this target there.

Reply:
Done